<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SmartSinePy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string># of entries</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amplitude</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Offset</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start angle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End angle</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inputs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copy">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy to Clipboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calculate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plotLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Entries per line</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_slider">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
